<launch>
    <node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk" output="screen">
    <!-- node parameters -->
    <param name="serial_name" type="string" value="/dev/ttyTHS1"/>
    <param name="baud_rate" type="int" value="230400" />
    <param name="app_id" type="int" value="1058906"/>
    <param name="app_version" type="int" value="1"/>
    <param name="align_time" type="bool" value="false"/>
    <param name="enc_key" type="string" value="ba2750e3908c80acabd86eaee63c7d9cbf7798713f39bb431780d00ed66c5f23"/>
    <param name="use_broadcast" type="bool" value="false"/>
    </node>
</launch>
